cli/cmd/testdata/install_default.golden

```diff
@@ -546,4 +546,278 @@ data:
       - source_labels: [__meta_kubernetes_pod_container_name]
         action: replace
         target_label: job
+
+### Grafana ###
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: grafana
+  namespace: conduit
+  labels:
+    app: grafana
+    conduit.io/control-plane-component: grafana
+  annotations:
+    conduit.io/created-by: conduit/cli undefined
+spec:
+  type: ClusterIP
+  selector:
+    app: grafana
+  ports:
+  - name: http
+    port: 3000
+    targetPort: 3000
+
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    conduit.io/created-by: conduit/cli undefined
+  creationTimestamp: null
+  labels:
+    app: grafana
+    conduit.io/control-plane-component: grafana
+  name: grafana
+  namespace: conduit
+spec:
+  replicas: 1
+  strategy: {}
+  template:
+    metadata:
+      annotations:
+        conduit.io/created-by: conduit/cli undefined
+        conduit.io/proxy-version: undefined
+      creationTimestamp: null
+      labels:
+        app: grafana
+        conduit.io/control-plane-component: grafana
```

Comment:
Do we really want to expose this with "graphana" branding everywhere? Maybe it would be better to name it something like "graphs" or "health" or something? I don't have strong feelings on this; it just seems odd that we'd effectively promise that this is always going to be grafana and never a different implementation.

Reply:
I think mention of `grafana` helps users to know what's included, similar to our mention of `prometheus`. If we ever swapped either of these components out, I think it would be a substantial enough change to warrant it affecting the surface area here.